CONTRIBUTING.md

```diff
@@ -48,16 +60,40 @@ To make sure if the import lists are stored correctly:
 
 Don't forget to add a short description of your change in the [CHANGELOG](CHANGELOG.md)
 
-### Publishing (by maintainer)
+### Create a Pull Request
+
+Once you've made your changes and ensured they pass all tests and quality checks, it's time to submit them for review. Follow these steps to create a pull request:
+
+1) Commit your changes:
+```bash
+Copy code
+git add .
+git commit -m "Brief description of your changes"
+```
+2) Push your changes to your forked repository:
+```bash
+Copy code
+git push origin master
+```
+3) Visit your forked repository on GitHub. You should see a message prompting you to create a pull request based on the changes you just pushed.
+4) Click on the "Compare & pull request" button to start creating the pull request. Ensure the base repository is set to plamere/spotipy and the base branch is master.
+5) Provide a descriptive title and summary of your changes in the pull request description.
+6) Click "Create pull request" to submit your changes for review.
+
+### Review Process
+
+Once you've submitted your pull request, project maintainers will review your changes. They may provide feedback or request additional changes. Be responsive to feedback and iterate on your changes as needed until they are approved.
+
+## Publishing (by maintainer)
 
  - Bump version in setup.py
  - Bump and date changelog
  - Add to changelog:
 
        ## Unreleased
-       Add your changes below.
 
        ### Added
+       - Replace with changes
```

Comment:
If you're going to remove the line above, you could add the `- Replace with changes` to Fixed and Removed as well.